src/core/InformationPanel/InformationPanelContent.tsx

```diff
@@ -0,0 +1,114 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import {
+  CommonProps,
+  useMergedRefs,
+  useResizeObserver,
+  useTheme,
+} from '../utils';
+import '@itwin/itwinui-css/css/information-panel.css';
+
+export type InformationPanelContentProps = {
+  /**
+   * Maximum width of the panel below which layout will shift from two columns to one,
+   * i.e. label and input will display on separate lines for widths lower than this value.
+   *
+   * If set to 0, the layout will always be two columns.
+   * If set to `Number.POSITIVE_INFINITY`, the layout will always be one column.
+   *
+   * @default 0
+   */
+  responsiveBreakpoint?: number;
```

Comment:
Need feedback on this part

Reply:
oh wow...
Couldnt we somehow give this control to the user, so he handles, when something happens to content??

Reply:
You mean like prop to toggle 2 columns vs 1?

i doubt users will add resize observers to toggle it responsively

Reply:
Lets rename this prop to something like `displayType`, `styleType` or something from that family to basically give option `block` or `inline`. With that, we just apply `iui-inline` class when needed. And then this example with using our custom hook we can give in both: stories and @example above this component. This will give us more flexibility and we leave user to what ever custom logic he might want to have to arrange this layout.
There is still a risk of having some custom layout, which we wont support and will have issues, but for now I think that would be a better solution than having this custom prop.
And lets merge this, cause its just staying for too long.

Reply:
It's an internal hook. Sounds like a bad idea to show it in examples.

Reply:
Ok, dont show then. Just rename the prop and lets merge this this year somehow..

Reply:
ok changed